CODEOWNERS

```diff
@@ -1 +1,2 @@
 *  @jriffs
+*  @lawblaze
```

Comment:
you don't really need to add this, also, since you don't have write permission for the repo, this will throw an error